### ISSUE TITLE
Bug: IP address mapping display

### ISSUE THREAD

**Issue by shawnkhall** [Steppschuh/RemoteControlServer#98]
It appears that the server actually maps to all available IP addresses, not only the one that's displayed. It would be nice if instead of displaying just the first IP it discovered that it would either display all, or cycle through all valid IP addresses when you click on it. The first IP in my environment, for example, is for virtualbox, so is useless when trying to map from my phone. The Autodiscovery from my phone worked great for identifying the server anyway, but it would have been better if I could visually inspect the IP address for validity since many of my devices are similarly named.

**Comment by Steppschuh** [comment]
That's true, this is an issue right now. You can see all IP addresses in the Server Log, but most users won't notice that. I'll think of a way for improving that,  thanks for reporting.
